File: modul-4/Soal-5/PRAK405-2310817110013-Randy_Febrian.c
#include <stdio.h>

int main(){
    int jumlah, kelipatan;
    int hasil = 0;
    scanf("%d %d", &jumlah, &kelipatan);

    for(int i = 1; i <= jumlah; i++){
        hasil = hasil + (i * kelipatan); 
        for(int j = i; j >= 1; j--){
            printf("(%d * %d)", j, jumlah);
            if(j > 1){
                printf(" + ");
            }
        }
        printf(" = %d\n", hasil);
    }
    return 0;
}

File: modul-4/Soal-5/PRAK405-2310817110013-Randy_Febrian.py
jumlah, kelipatan = map(int, input().split())
hasil = 0

for i in range(1, jumlah + 1):
    hasil += i * kelipatan
    for j in range(i, 0, -1):
        print("(%d * %d)" % (j, kelipatan), end=" + " if j > 1 else "")  # Print each term
    print("= %d" % hasil)